src/core/fonts.js

```diff
@@ -133,7 +134,53 @@ function adjustWidths(properties) {
   properties.defaultWidth *= scale;
 }
 
-function adjustToUnicode(properties, builtInEncoding) {
+function adjustTrueTypeToUnicode(properties, isSymbolicFont, nameRecords) {
+  if (properties.isInternalFont) {
+    return;
+  }
+  if (properties.hasIncludedToUnicodeMap) {
+    return; // The font dictionary has a `ToUnicode` entry.
+  }
+  if (properties.hasEncoding) {
+    return; // The font dictionary has an `Encoding` entry.
+  }
+  if (properties.toUnicode instanceof IdentityToUnicodeMap) {
+    return;
+  }
+  if (!isSymbolicFont) {
+    return; // A non-symbolic font should default to `StandardEncoding`.
+  }
+
+  // Try to infer if the fallback encoding should really be `WinAnsiEncoding`.
+  if (properties.defaultEncoding === WinAnsiEncoding) {
+    return;
+  }
+  for (const r of nameRecords) {
+    if (!(r.platform === 3 && r.encoding === 1 && r.language === 0x409)) {
```

Comment:
nit: add a comment with https://learn.microsoft.com/en-us/typography/opentype/spec/name#windows-language-ids to explain the 0x409.

Reply:
Well, I was just copying from here: https://github.com/mozilla/pdf.js/blob/d8d5545e03d2c7a17b414a65684a745389f2a614/src/core/fonts.js#L2213-L2219

Reply:
I've added to two new helper functions, with documentation links.